src/python/pants/backend/project_info/peek.py

```diff
@@ -47,6 +53,16 @@ class PeekSubsystem(Outputting, GoalSubsystem):
         help="Whether to leave off values that match the target-defined default values.",
     )
 
+    include_dep_rules = BoolOption(
+        default=False,
+        help=softwrap(
+            """
+            Whether to include `_dependencies_rules`, `_dependents_rules` and `_applicable_dep_rules`
```

Comment:
Why the underscore prefix?

Reply:
It's me trying to be cautious to not get a key conflict with a target field name (in the future). This flat namespace that is the `peek` data is making titchy. Hence https://github.com/pantsbuild/pants/pull/18383#issuecomment-1448999983

Reply:
i.e. I propose we adjust the peek schema, and with that we can also get rid of these underscores; but that's for another PR ;)